projects/backend/controllers/render.ts

```diff
@@ -164,6 +164,7 @@ const processArticleRendering = async (
     internalPageCode: number,
     furniture: PublishedFurniture,
     theme: Theme | null,
+    isPreview: boolean = false,
```

Comment:
Like the linter comment below says if we do `isPreview = false` typescript will automatically infer the type